reflred/steps.py

```diff
@@ -1405,16 +1405,21 @@ def abinitio_footprint(data, Io=1., width=None, offset=0.):
     offset (float:mm) : offset of the center of rotation of the sample in
     the direction of the beam, toward the detector.
 
+    source_divergence {source divergence (deg, full width)} (float?) : override the
+    source divergence found in the data.  Fill in this value if your divergence
+    is known and was not entered on load
+
     **Returns**
 
     outputs (refldata): footprint-corrected data
 
-    2016-09-02 Paul Kienzle
+    | 2016-09-02 Paul Kienzle
+    | 2022-02-08 Brian Maranville add source_divergence input
     """
     from .footprint import apply_abinitio_footprint
 
     data = copy(data)
-    apply_abinitio_footprint(data, Io, width, offset)
+    apply_abinitio_footprint(data, Io, width, offset, source_divergence)
```

Comment:
`steps.abinitio_footprint` is a user interface function. This is the appropriate place to convert from external units to internal units (i.e., FWHM to 1-σ).

Reply:
makes sense